app/models/rating.rb

```diff
@@ -110,30 +108,9 @@ class Rating < ActiveRecord::Base
 
   before_validation :set_data
 
-  validates :rateable, presence: true
-  validates :user_id, presence: true, uniqueness: { scope: [:rateable_type, :rateable_id] }
-
-  class << self
-    def user_names
-      User.includes(:roles).where('roles.name = ?', 'reviewer').pluck(:name)
-    end
-
-    def excluding(names)
-      joins(:user).where('users.name NOT IN (?)', names)
-    end
+  validates :user_id, uniqueness: { scope: :application_id }
 
-    def by(user)
-      where(user_id: user.id)
-    end
-
-    def for(type, id)
-      where(rateable_type: type, rateable_id: id)
-    end
-  end
-
-  def woman?
-    data[:is_woman] == 1
-  end
```

Comment:
Bouble check:  this criterium is not in use anymore?

Reply:
I couldn't find any usage of it in the code. Wasn't there a bigger change to the way people would specify their gender?

Reply:
If it is not in the code, than it is not in the rating #thingsRailsGirlsSay 😂